runners/flink/job-server/build.gradle

```diff
@@ -23,14 +24,27 @@ applyJavaNature(
   },
 )
 
+/*
+ * We need to rely on manually specifying these evaluationDependsOn to ensure that
+ * the following projects are evaluated before we evaluate this project. This is because
+ * we are attempting to reference the "sourceSets.test.output" directly.
+ */
+evaluationDependsOn(":beam-runners-flink_2.11")
```

Comment:
You reference `:beam-sdks-java-core` and `:beam-runners-core-java` sourceSets.test.output directly and not `:beam-runners-flink_2.11`